pkg/tsdb/mysql/mysql_test.go

```diff
@@ -150,11 +150,11 @@ func TestMySQL(t *testing.T) {
 			require.Equal(t, "def", *frameOne.Fields[2].At(0).(*string))
 			require.Equal(t, int32(1), frameOne.Fields[3].At(0).(int32))
 			require.Equal(t, int16(10), frameOne.Fields[4].At(0).(int16))
-			require.Equal(t, int64(100), frameOne.Fields[5].At(0).(int64))
-			require.Equal(t, int32(1420070400), frameOne.Fields[6].At(0).(int32))
+			require.Equal(t, int64(100), *(frameOne.Fields[5].At(0).(*int64)))
+			require.Equal(t, int64(1420070400), *(frameOne.Fields[6].At(0).(*int64)))
 			require.Equal(t, 1.11, *frameOne.Fields[7].At(0).(*float64))
 			require.Equal(t, 2.22, *frameOne.Fields[8].At(0).(*float64))
-			require.Equal(t, float32(3.33), frameOne.Fields[9].At(0).(float32))
+			require.Equal(t, float64(3.33), *(frameOne.Fields[9].At(0).(*float64)))
```

Comment:
If you look at the converter for this test, the converter returns a `*int64`, and has a `OutputFieldType` of `NullableInt64`.